sys/arm64/arm64/locore.S

```diff
@@ -665,7 +682,12 @@ start_mmu:
 
 	/* Load the exception vectors */
 	ldr	x2, =exception_vectors
+#ifdef MORELLO
+	cvtp	c2, x2
```

Comment:
```suggestion
	/* TODO: Remove when the kernel understands capabilities */
	cvtp	c2, x2
```
?

Reply:
All uses of cvtp have an implicit TODO to review later.